test/e2e/helm/kuma_helm_deploy_global_universal_mode_and_zone.go

```diff
@@ -0,0 +1,160 @@
+			)).
+			// Install(WaitService("kuma-system", "postgres-release-postgresql")). // this does not seem to work
+			// control plane crashes twice waiting for postgres to come up
+			Install(YamlK8s(`
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres
+  namespace: kuma-system
+type: Opaque
+stringData:
+  password: "mesh"
+`)).
+			Setup(c1)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = NewClusterSetup().
+			Install(Kuma(core.Global,
+				WithSkipDefaultMesh(true),
+				WithInstallationMode(HelmInstallationMode),
+				WithHelmReleaseName(releaseName),
+				WithCPReplicas(2),
+				WithHelmOpt("controlPlane.environment", "universal"),
+				WithHelmOpt("postgres.host", "postgres-release-postgresql"),
+				WithHelmOpt("postgres.port", "5432"),
+				WithHelmOpt("postgres.user", "mesh"),
+				WithHelmOpt("postgres.db", "mesh"),
+				WithHelmOpt("controlPlane.secrets[0].Secret", "postgres"),
+				WithHelmOpt("controlPlane.secrets[0].Key", "password"),
+				WithHelmOpt("controlPlane.secrets[0].Env", "KUMA_STORE_POSTGRES_PASSWORD"),
+				WithHelmOpt("controlPlane.config", `
+interCp:
+  catalog:
+    heartbeatInterval: 1s
+    writerInterval: 3s
+`),
+			)).
+			Install(MeshUniversal("default")).
+			Setup(c1)
+		Expect(err).ToNot(HaveOccurred())
+
+		global = c1.GetKuma()
+		Expect(global).ToNot(BeNil())
+
+		err = NewClusterSetup().
+			Install(Kuma(core.Zone,
+				WithSkipDefaultMesh(true),
+				WithInstallationMode(HelmInstallationMode),
+				WithHelmReleaseName(releaseName),
+				WithGlobalAddress(global.GetKDSServerAddress()),
+				WithHelmOpt("ingress.enabled", "true"),
+			)).
+			Install(NamespaceWithSidecarInjection(TestNamespace)).
+			Install(DemoClientK8s("default", TestNamespace)).
+			Install(testserver.Install()).
+			Setup(c2)
+		Expect(err).ToNot(HaveOccurred())
+
+		zone = c2.GetKuma()
+		Expect(zone).ToNot(BeNil())
+	})
+
+	E2EAfterAll(func() {
+		Expect(c2.DeleteNamespace(TestNamespace)).To(Succeed())
+		Expect(c1.DeleteKuma()).To(Succeed())
+		Expect(c2.DeleteKuma()).To(Succeed())
+		Expect(clusters.DismissCluster()).To(Succeed())
+	})
+
+	It("should deploy Zone and Global on 2 clusters", func() {
+		clustersStatus := api_server.Zones{}
+		Eventually(func() (bool, error) {
+			status, response := http_helper.HttpGet(c1.GetTesting(), global.GetGlobalStatusAPI(), nil)
+			if status != http.StatusOK {
+				return false, errors.Errorf("unable to contact server %s with status %d", global.GetGlobalStatusAPI(), status)
+			}
+			err := json.Unmarshal([]byte(response), &clustersStatus)
+			if err != nil {
+				return false, errors.Errorf("unable to parse response [%s] with error: %v", response, err)
+			}
+			if len(clustersStatus) != 1 {
+				return false, nil
+			}
+			return clustersStatus[0].Active, nil
+		}, "1m", "1s").Should(BeTrue())
+
+		// then
+		active := true
+		for _, cluster := range clustersStatus {
+			if !cluster.Active {
+				active = false
+			}
+		}
+		Expect(active).To(BeTrue())
```

Comment:
Instead of this, check if Mesh is synced to zone or check ZoneInsights. We should drop this API